.eslintrc.json

```diff
@@ -13,5 +18,8 @@
     "class-methods-use-this": [
```

Comment:
I noticed the tests are throwing a `import/no-extraneous-dependencies` error -- fortunately this is easy to fix, just add `"*.test.jsx"` inside the `devDependencies` object up above (add it after line 14 -- i can't commend directly on that line for some reason). This tells eslint:
- for the `import/no-extraneous-dependencies` rule,
- if it applies to imports from `devDependencies`,
- skip it for all files that match the `*.test.jsx` pattern (because technically tests aren't part of production code, so we don't need to treat their deps as prod deps)

As you might notice, we're already applying this rule to everything within the config directory (which is also not *technically* part of production code) as well.

Reply:
I've added it. I needed to tack on a "**" to make it "**/*.test.jsx" in order for it to work. I'm not totally sure why, since the one above it worked fine, but maybe that's because it was specifying a top-level directory that has a flat structure inside instead of nested directories?